ChangeLog

```diff
@@ -1,3 +1,22 @@
+Thu, 21 Mar 2024 08:50:54 +0100 IvozProvider Team <vozip+ivozprovider@irontec.com>
+
+    * IvozProvider 4.0.3 released
+
+    * Portals
+        - Fixed a bug while displaying terminal provisioning template test dialog
+        - Fixed a bug while displaying Domain filtering options
+        - Added missing ACL entries for Holiday Date ranges
+        - Invoice Numbers prefix is now optional
+        - Increased inactivity timeout in Active Calls to one hour
+        - Updated Active calls call direction icons
+
+    * Provisioning
+        - Fixed a bug that prevented running PHP code in terminal provisioning templates
+
+    * Proxies
+        - Fixed a bug with Cisco Phones BLF Subscriptions
```

Comment:
- Fix behind nat proxied UAC SUBSCRIBE routing

Reply:
What about using Fixed lke the other entries:  _Fixed behind NAT SUBSCRIBE routing being proxied_

Reply:
good